### PR TITLE
Metrics api

### PR DESCRIPTION
**Pull request by mark845163**

### CODE REVIEW

**Comment by rax845163** [comment]
@mamafun 
The generated code was affected by action of the folder changed from ai-service to aiservice.

File: operator/v1alpha1/server.proto [hunk 1]
```diff
@@ -21,8 +57,25 @@ message GetMetricsRequest {
     repeated LabelSelector conditions = 5;
 }
 
-message GetMetricsResponse {
-    repeated Result results = 1;
+message GetMetricResponse {
+    MetricResult result = 1;
```
Comment (mamafun) [inline operator/v1alpha1/server.proto]: What if GetMetricRequest() queries contains multiple targets (selected by the LabelSelector argument) ?
Do we need a "repeated" decorator here?

Comment (rax845163) [reply operator/v1alpha1/server.proto]: @mamafun 
Yes, please review the commitment, it has been revised.

File: operator/v1alpha1/server.proto [hunk 2]
```diff
@@ -21,8 +57,25 @@ message GetMetricsRequest {
     repeated LabelSelector conditions = 5;
 }
 
-message GetMetricsResponse {
-    repeated Result results = 1;
+message ListMetricsResponse {
+    repeated MetricResult metrics = 1;
```
Comment (yamlin) [inline operator/v1alpha1/server.proto]: One more question about response. Would you like to support pagination in the future? Or just fix the number of metrics?

**Pull request merged**